### PR TITLE
Task #118324 feat: Load font-awesome.min.css in backend

### PR DESCRIPTION
**Pull request by sneha27**

### CODE REVIEW

File: tjreports/administrator/tjreports.php [hunk 1]
```diff
@@ -34,14 +34,15 @@
 }
 
 $document = JFactory::getDocument();
-
 // Load js assets
 $tjStrapperPath = JPATH_SITE . '/media/techjoomla_strapper/tjstrapper.php';
```
Comment (irfanhanfi123) [inline tjreports/administrator/tjreports.php]: @sneha27 Please add this font awesome without strapper and remove this strapper code.

Comment (manojLondhe) [reply tjreports/administrator/tjreports.php]: Are there any other files loaded using strapper?

Comment (irfanhanfi123) [reply tjreports/administrator/tjreports.php]: @manojLondhe No, we are not using this to load any files.
@sneha27 Also check site tjreports.php and remove the same strapper code from there as well.

**Comment by irfanhanfi123** [comment]
@sneha27 fonts folder is missing.

**Comment by manojLondhe** [comment]
@irfanhanfi123 @sneha27 have you checked if this newly added folder gets copied in the installation process?

**Comment by irfanhanfi123** [comment]
@manojLondhe actually font-awesome folder is in the CSS folder which is already in the xml. So it should be copied without issue. should we move that outside CSS folder?

**Pull request merged**